lib/3scale_toolbox/commands/service_command/service_show.rb

```diff
@@ -0,0 +1,99 @@
+module ThreeScaleToolbox
+  module Commands
+    module ServiceCommand
+      class ServiceShowSubcommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+
+        def self.command
+          Cri::Command.define do
+            name        'show'
+            usage       'show'
+            summary     'service show'
+            description "Show the information of a service"
+            runner ServiceShowSubcommand
+
+            param   :source
+            param   :service_id_or_system_name
+          end
+        end
+
+        def run
+          source = arguments[:source]
+          service_id_or_system_name = arguments[:service_id_or_system_name]
+          client = threescale_client(source)
+
+          service = service_data_from_id_or_system_name(client, service_id_or_system_name)
+          if !service
+            abort "Service #{service_id_or_system_name} not found"
+          end
+          print_service_data(service, SERVICE_FIELDS_TO_SHOW)
+        end
+
+        private
+
+        SERVICE_FIELDS_TO_SHOW = %w[
+          id name state system_name end_user_registration_required
+          backend_version deployment_option support_email description
+          created_at updated_at
+        ]
+
+        def param_natural_number?(param)
```

Comment:
There is `ThreeScaleToolbox::Helper` module for this kind of methods

Reply:
Finally we'll let the API raise a exception if incompatible